server/pulp/server/controllers/distributor.py

```diff
@@ -6,7 +6,7 @@
 import celery
 
 from pulp.common import tags
-from pulp.common.error_codes import PLP0002, PLP0003
+from pulp.common.error_codes import PLP0002
```

Comment:
Undo this change as well. I think we should not add any changes to server/* even to make it flake8. It might imply that this code is not dead in the 3.0-dev line.